packages/drivers/local-driver/src/localDocumentStorageService.ts

```diff
@@ -82,6 +94,13 @@ export class LocalDocumentStorageService implements IDocumentStorageService {
 		summary: ISummaryTree,
 		context: ISummaryContext,
 	): Promise<string> {
+		if (context.referenceSequenceNumber === 0) {
+			await createDocument(
+				this.localDeltaConnectionServer,
+				this.resolvedUrl as IFluidResolvedUrl,
```

Comment:
we should avoid casting. in this case, call ensureFluidResolvedUrl(resolvedUrl ) right before createDocument.